class-pikpay.php

```diff
@@ -1859,7 +1848,6 @@ public function get_sr_translation()
 
         return $lang;
     }
-
```

Comment:
@KipepeoYaMachungwa a lot of formatting changes, it's hard to figure out what changed.